modules/rubiconBidAdapter.js

```diff
@@ -1001,6 +1004,22 @@ function applyFPD(bidRequest, mediaType, data) {
   }
 }
 
+function addDesiredSegtaxes(bidderRequest, target) {
+  if (rubiConf.readTopics === false) {
+    return;
+  }
+  let iSegments = [1, 2, 5, 6, 507].concat(rubiConf.sendUserSegtax?.map(seg => Number(seg)));
+  let vSegments = [4, 508].concat(rubiConf.sendSiteSegtax?.map(seg => Number(seg)));
```

Comment:
I think these will add `undefined` to the arrays

To get around this can just add `|| []` I think at end of map so it skips it.

Other ways to handle also.

Think adding `undefined` to these arrays is prob not a big deal, but is unnecessary. 

EX:

```js
iSegments = [1, 2, 5, 6, 507].concat(rubiConf.sendUserSegtax?.map(seg => Number(seg)) || []);
```